biscuit-auth/src/token/third_party.rs

```diff
@@ -161,6 +162,7 @@ impl ThirdPartyRequest {
 ///
 /// this must be integrated with the token that created the [`ThirdPartyRequest`]
 /// using [`Biscuit::append_third_party`](crate::Biscuit::append_third_party)
+#[derive(Debug, Default)]
```

Comment:
what does `Default` mean in this context?

Reply:
sorry, wrong push repo :'( reverting it